### ISSUE TITLE
Do "crossings" functions work properly when value lands on median/mean while crossing?

### ISSUE THREAD

**Issue by bminor-wsu** [WSU-CASAS/AL#11]
For the `features.py` `zero_crossings()` and `mean_crossings()` functions, when a value is exactly equal to the median/mean, we set `rel = 0` (the previous value sign indicator is set to zero).

But what happens if we have one point below the median, one at the median, and one above it, in that order (or vice versa)? Then we'd have:

1. `x < median` -> `rel = -1`
2. `x == median` -> `rel = 0`
3. `x > median`, `rel !< 0` -> don't increment `count`

In other words, the values cross the median, but we do not increment the `count`. But, it should be incremented since this is technically crossing the median/mean?

I think a simple solution to fix this would be to just remove the `else: rel = 0` lines as that would leave the sign value at whatever it was before.
